React/Base/RCTMultipartStreamReader.m

```diff
@@ -76,7 +76,7 @@ - (void)emitProgress:(NSDictionary *)headers
   // Throttle progress events so we don't send more that around 60 per second.
   CFTimeInterval currentTime = CACurrentMediaTime();
 
-  NSUInteger headersContentLength = headers[@"Content-Length"] != nil ? [headers[@"Content-Length"] unsignedIntValue] : 0;
```

Comment:
`unsignedIntegerValue` maybe?

Reply:
unfortunately no, both `unsignedIntValue` and `unsignedIntegerValue` breaks (at runtime)

Reply:
I think that the statement can be changed to
NSInteger headersContentLength = headers[@"Content-Length"] != nil ? [headers[@"Content-Length"] intValue] : 0; so the value didn't have to be cast.